src/fdc3/dotnet/DesktopAgent/test/MorganStanley.ComposeUI.DesktopAgent.Tests/EndToEndTests.cs

```diff
@@ -257,6 +257,8 @@ public async Task FindIntentReturnsAppIntent()
         var result = resultBuffer!.ReadJson<FindIntentResponse>(_options);
         result.Should().NotBeNull();
         result.Should().BeEquivalentTo(expectedResponse);
+
+        await _moduleLoader.StopModule(new StopRequest(instance.InstanceId));
```

Comment:
These will not be called if the assertion fails, it should either be in a try-finally block or better yet make the test class disposable and do the cleanup there - that would probably spare a lot of boilerplate code as well

Reply:
We already using IAsyncLifetime which exposes that DisposeAsync will be called after every tests. I remove the unnecessary/redundant and wrong calls :)